### PR TITLE
chore(mender-gateway): Add download instructions for on-prem environments

### PR DESCRIPTION
**Pull request by MaciejTe**
chore(mender-gateway): Add download instructions for on-prem environments

Changelog: None
Ticket: MEN-6006
Signed-off-by: Maciej Tomczuk <maciej.tomczuk@northern.tech>

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

**Comment by lluiscampos** [comment]
I'll let @MuchoLucho do the final review.

File: 01.Get-started/06.Mender-Gateway/docs.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ you will connect to the Mender Gateway.
 
 The first step is installing the Mender Gateway into your Raspberry Pi device.
 
+[ui-tabs position="top-left" active="0" theme="lite" ]
+[ui-tab title="hosted"]
+<!--AUTOMATION: ignore -->
```
[the remark this answers is not in the document]
Comment (MaciejTe) [reply 01.Get-started/06.Mender-Gateway/docs.md]: Done!

**Review by TheYoctoJester — APPROVED** [review]
Content wise fine by me.

Did not verify the markup, e.g. not build it.

**Comment by lluiscampos** [comment]
@mender-test-bot cherry-pick to:
* 3.4.x
* 3.3.x

**Pull request merged**